<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#LnAddr[state = &quot;0&quot;] {
	background-color: rgb(237, 51, 59);
}

#LnAddr[state = &quot;1&quot;] {
	background-color: rgb(38, 162, 105);
}

#GrpIp {
	background-color: rgb(192, 191, 188);
	color: rgb(0, 0, 0);
}

#GrpClient{
	background-color: rgb(192, 191, 188);
	color: rgb(0, 0, 0);
}

#LblMicro {
	color: rgb(0, 0, 0);
}

#LblFrom {
	color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="LblTitle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BABEL</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>521</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="GrpIp">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Connection information</string>
       </property>
       <widget class="QLabel" name="LblAddr">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Address :</string>
        </property>
       </widget>
       <widget class="QLabel" name="LblPort">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LnAddr">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMask">
         <string>000.000.000.000;_</string>
        </property>
        <property name="state" stdset="0">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="SpnPort">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65534</number>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnConnect">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QPushButton" name="RecBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>180</y>
          <width>87</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
       <widget class="QComboBox" name="CBInputs">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>140</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="LblMicro">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>140</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
       <widget class="QLabel" name="LblOutput">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>80</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
       <widget class="QComboBox" name="CBOutputs">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>80</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnLst">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>180</y>
          <width>87</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Listen</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GrpClient">
       <property name="title">
        <string>Connect to client : </string>
       </property>
       <widget class="QListWidget" name="LstUsers">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnCall">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>87</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Call</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnAccept">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>87</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnRefuse">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>87</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Refuse</string>
        </property>
       </widget>
       <widget class="QLabel" name="LblFrom">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>You're being called by : </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LnMessage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>258</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="LstConsole">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(192, 191, 188);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnClear</sender>
   <signal>clicked()</signal>
   <receiver>LstConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
